hash/ibm5170_cdrom.xml

```diff
@@ -4014,6 +4014,65 @@ https://mametesters.org/view.php?id=9334
 		</part>
 	</software>
 
+	<!-- Windows 3.1/95 -->
+	<!-- http://redump.org/disc/114360/ -->
+	<software name="fulltilt">
+		<description>Full Tilt! Pinball (version 1.1)</description>
+		<year>1996</year>
+		<publisher>Maxis</publisher>
+		<info name="developer" value="Cinematronics" />
+		<info name="release" value="19960319" />
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="full tilt pinball 1.1" sha1="c9ab5a92a673924ae341fb6f671715ca42bf6f8c" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Windows 3.1/95 -->
+	<!-- http://redump.org/disc/81828/ -->
+	<software name="fulltiltoem1" cloneof="fulltilt">
+		<description>Full Tilt! Pinball (version 1.1) (Scholastic Edition)</description>
+		<year>1996</year>
+		<publisher>Maxis</publisher>
+		<info name="developer" value="Cinematronics" />
+		<info name="release" value="19960319" />
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="full tilt pinball 1.1 scholastic" sha1="ff4ad114486f372502359271b5e28bae1975d884" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Windows 3.1/95 -->
+	<!-- http://redump.org/disc/15320 -->
+	<software name="fulltiltoem2" cloneof="fulltilt">
+		<description>Full Tilt! Pinball (version 1.1) (Thrustmaster OEM)</description>
```

Comment:
This may be a partial: it may use the Wizzard Pinball controller, check readme.

Reply:
Any unsupported/badly supported optional device needs a `supported="partial"`, obviously with some common sense on what said device does. 
afaik the Wizzard Pinball attach to joy port with pressure sensitive buttons, akin on how a real pinball machine behaves. For more info: https://www.youtube.com/watch?v=-UHVFohpCqU